src/Evolution/Systems/Cce/GaugeTransformBoundaryData.hpp

```diff
@@ -848,4 +848,29 @@ struct InitializeGauge {
       gsl::not_null<Scalar<SpinWeighted<ComplexDataVector, 0>>*> omega,
       size_t l_max);
 };
+
+/*!
+ * \brief Compute a worldtube constraint for Klein-Gordon Cce
+ *
+ * \details Both Cauchy and characteristic systems evolve the scalar field
+ * independently. The results have to be consistent at the worldtube. The
+ * difference (`Tags::KleinGordonWorldtubeConstraint`) between the Cauchy
+ * worldtube data `Tags::BoundaryValue<Tags::KleinGordonPsi>` and the evolved
+ * volume data `Tags::KleinGordonPsi` (at the worldtube)
+ * indicates the accuracy of the simulation.
+ */
+struct ComputeKGWorldtubeConstraint {
```

Comment:
This doesn't appear to have anything to do with gauge transformations, so it should probably go in a different file.

Reply:
I've moved the mutator to a new file.